CONTRIBUTING.md

```diff
@@ -55,37 +55,37 @@ Thank you! We'll try to get back to you as soon as possible.
 1. Install Rust using [rustup], which allows you to easily switch between Rust
    versions. Diesel currently supports Rust Stable, Nightly, Rust Beta.
 
-   If you want to run Diesel's test suite with _all_ supported features (extra
+   To run Diesel's test suite with _all_ supported features (extra
    lints and compiletest), you should use the same nightly version as Diesel's
-   continuous integration. You can find it by looking for a line like
+   continuous integration. Find it by looking for a line like
    `rust: nightly-2017-06-06` in the `.travis.yml` file. You can install and
    set a custom nightly version for a project using
    `rustup override add nightly-2017-06-06`.
 
 2. Install the system libraries needed to interface with the database systems
    you wish to use.
 
-   These are the same as when compiling diesel. In general, it is a good idea
-   to have _all_ drivers installed so you can run all tests locally.
+   These are the same as when compiling Diesel. It's generally a good idea
+   to install _all_ drivers so you can run all tests locally.
 
-   *Shortcut:* On macOS, you don't need to install anything to work with SQLite
-   and for PostgreSQL you'll only need the server (`libpq` is installed by
-   default). So, to get started, `brew install postgresql mysql` and follow the
+   *Shortcut:* On MacOS, you don't need to install anything to work with SQLite.
```

Comment:
Apple writes macOS with a lowercase 'm' :)

Reply:
Oh wow, thanks for that catch!